### PR TITLE
Update FastText countDownloads

### PR DESCRIPTION
**Pull request by kenhktsui**
Add query file for FastText library according to [here](https://huggingface.co/docs/hub/en/models-download-stats).  
Most FastText model is in format of [bin](https://fasttext.cc/docs/en/python-module.html) so a wildcard of "*.bin" is added.

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:168]: For what I can quickly see on https://huggingface.co/models?library=fasttext&sort=trending, fasttext models are even saved as `model.bin`. Therefore, I would recommend to count downloads on this file specifically. 2 advantages:
- foster consistent naming on the Hub for fasttext models
- exact terms are more efficient than wildcards

```suggestion
		countDownloads: {
			term: { path: "model.bin" },
		},
```

**Review by Wauplin — COMMENTED** [review]
Thanks for the PR @kenhktsui! Added a small comment on the ES query :)

File: packages/tasks/src/model-libraries.ts
Comment (kenhktsui) [reply packages/tasks/src/model-libraries.ts]: Thanks @Wauplin! I agree with you suggestion and have committed it.

**Review by Wauplin — APPROVED** [review]
Looks good to me! 

Since fasttext was added quite some time ago I'd prefer a second review from @osanseviero / @julien-c to confirm

**Review by julien-c — APPROVED** [review]
lgtm, maybe also add this conventional name to the fastText documentation if they still maintain it?

File: packages/tasks/src/model-libraries.ts
Comment (osanseviero) [inline packages/tasks/src/model-libraries.ts:167]: Some repos don't use `model.bin`, but other names

* `model_v1.bin` https://huggingface.co/cis-lmu/glotlid/tree/main 
* `debate2vec.bin` https://huggingface.co/Hellisotherpeople/debate2vec 
* and [so on](https://huggingface.co/NbAiLab/nb-nordic-lid/blob/main/nb-nordic-lid.bin))

Maybe better to use a wildcard here, WDYT?

Comment (kenhktsui) [reply packages/tasks/src/model-libraries.ts:167]: > Some repos don't use `model.bin`, but other names
> 
> * `model_v1.bin` https://huggingface.co/cis-lmu/glotlid/tree/main
> * `debate2vec.bin` https://huggingface.co/Hellisotherpeople/debate2vec
> * and [so on](https://huggingface.co/NbAiLab/nb-nordic-lid/blob/main/nb-nordic-lid.bin))
> 
> Maybe better to use a wildcard here, WDYT?

Thanks for your comment @osanseviero 

Collating all the comments:
- Some popular fastText models are not using model.bin as @osanseviero suggested
- the naming convention fastText documentation is not deployed (and is not within our control) so users are not aware of it
- fastText model (while there is still a lot) is not a majority in the hub, so the ES query efficiency (due to wildcard vs exact) might not be a major concern to ES

Perhaps, we could go for the wildcard query for now? @Wauplin @julien-c wdyt?

Comment (Wauplin) [reply packages/tasks/src/model-libraries.ts:167]: Let's go with the wildcard then, thanks for the input @osanseviero @kenhktsui!

Comment (kenhktsui) [reply packages/tasks/src/model-libraries.ts]: Thanks @Wauplin. I have pushed the commit.

**Review by pcuenca — APPROVED** [review]
lgtm after the comments

File: packages/tasks/src/model-libraries.ts
Comment (julien-c) [reply packages/tasks/src/model-libraries.ts]: personally i think it's good to enforce some level of filename convention. That's what we do for other libraries too.

But up to you

Comment (julien-c) [reply packages/tasks/src/model-libraries.ts]: i.e., why even enforce `.bin` file extensions then? We could set `*` as wildcard then?

Comment (Wauplin) [reply packages/tasks/src/model-libraries.ts]: Sorry for the conflicting guidelines @kenhktsui :smile: 

Let's merge it as is with the `*.bin` wildcard.

**Pull request merged**
